examples/http.rs

```diff
@@ -0,0 +1,116 @@
+use std::collections::HashMap;
+
+use anyhow::Result;
+use hyper::header::CONTENT_TYPE;
+use serde::{Deserialize, Serialize};
+
+use esp_idf_svc::hal::prelude::Peripherals;
+use esp_idf_svc::log::EspLogger;
+use esp_idf_svc::wifi::{BlockingWifi, EspWifi};
+use esp_idf_svc::{eventloop::EspSystemEventLoop, nvs::EspDefaultNvsPartition};
+
+use lynx_embedded::{reqwesp, wifi as espWifi};
+
+#[allow(dead_code)]
+#[derive(Deserialize, Debug)]
+struct Anything {
+    pub data: String,
+    pub json: Option<Account>,
```

Comment:
Is this for optional?

Reply:
Yeah, an [`Option`](https://doc.rust-lang.org/rust-by-example/std/option.html) is an enum similar to `Result`. It yields either `Some(value)` or `None`. This is the Rust equivalent to how you would use `null` in other languages.

In this case, if the json field in the response contains `null`, it will deserialize to `None`. Otherwise it will deserialize to `Some(json)` where `json` is an `Account` struct.